### PR TITLE
Fix md5sum command not found issue on mac

### PR DESCRIPTION
**Pull request by vikramdattu**
The PR fixes and issue where some systems viz., mac do not have `md5sum` command.
It uses alias `md5 -r` for the same which bash doesn't expand.

Check for the same and use `md5 -r` instead.

### CODE REVIEW

**Review by bhack — APPROVED** [review]

**Review by advaitjain — APPROVED** [review]
Thanks for the fix. No action needed on the current PR, but I have filed https://github.com/tensorflow/tensorflow/issues/46020 for a more general fix.

Feel free to take that on if you'd like. Else I will pick it up at some point in time.

**Comment by bhack** [comment]
@rthadur It happens again. The butler rebuilds after the 2nd approval also without commits between approvals as we discussed some months ago at: https://github.com/tensorflow/tensorflow/pull/43754#issuecomment-704426064 
Any news?

**Pull request merged**
